doc/development/guide/pullrequests.rst

```diff
@@ -1,74 +1,58 @@
 Submitting a pull request
 =========================
 
-Before submitting a pull request, please make sure the following is done:
 
-* **All new features must include a unit test.** If you've fixed a bug or added
-  code that should be tested, add a test to the ``tests`` directory.
+Creating a Pull Request
+-----------------------
 
-  PennyLane uses pytest for testing; common fixtures can be found in the ``tests/conftest.py``
-  file.
+When you are ready, go ahead and click on "New Pull Request" on the `pull request page <https://github.com/PennyLaneAI/pennylane/pulls>`_.  On the next page, you need to select your fork and branch to merge into our master branch.
 
-* **All new functions and code must be clearly commented and documented.**
+On the next page, you need to specify the name of the PR and write an
+introductory description. We have a PR template with a checklist that can help
+you determine what to say. Please be as detailed as possible, and assume the reader does not have extensive background knowledge of the issue at hand.
 
-  Have a look through the source code at some of the existing function docstrings---
-  the easiest approach is to simply copy an existing docstring and modify it as appropriate.
+If the PR is in response to an Issue, please `link the issue <https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue>`_ with the relevant keywords.
 
-  If you do make documentation changes, make sure that the docs build and render correctly by
-  running ``make docs``. See our :doc:`documentation guidelines <documentation>` for more details.
+Once you create the PR, you can also view the status of the *Continuous Integration* (CI) checks for things like tests, code quality, and documentation rendering.
 
-* **Ensure that the test suite passes**, by running ``make test``.
+Continuous Integration Checks
+-----------------------------
 
-* **Make sure the modified code in the pull request conforms to the PEP8 coding standard.**
+Once a PR is opened on the repository, GitHub runs a series of checks on the
+branch.  All tests need to pass (green check) for the PR to get approved and
+merged. Sometimes, a check can be troublesome, especially test coverage,
+in which case designated core developers can override and merge anyway.
 
-  The PennyLane source code conforms to `PEP8 standards <https://www.python.org/dev/peps/pep-0008/>`_.
-  Before submitting the PR, you can autoformat your code changes using the
-  `Black <https://github.com/psf/black>`_ Python autoformatter, with max-line length set to 100:
+* **Formatting check**: This check makes sure `black` (described above) would not change any code.
```

Comment:
It looks like Black is not described above?

In any case, I would mention:

- how to install black (`pip install black`)
- that the `pennylane` and `tests` folder should both be blacked
- that the line length is 100

The last two can be done by giving the full command,
```
black -l 100 pennylane tests
```

Reply:
Thanks for catching this. It had been above before I switched ordering.